scripts/gen_release_requirements.py

```
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+"""生成发布版 requirements.txt（确定性裁剪 pyside6/pyside6-addons）。
+
+uv.lock 是依赖的唯一来源（AGENTS.md）。本脚本从 uv.lock 导出 Windows 平台依赖清单，
+再裁剪 pyside6 / pyside6-addons（qfluentwidgets 依赖 PySide6 元包引入，含
+QtWebEngine 约 200MB），发布包只保留 pyside6-essentials。配合 pyappify.yml 的
+pip_args "--no-deps"，用户端 pip 只按本清单精确安装，不再拉取 QtWebEngine。
+
+    生成并覆盖 requirements.txt：
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)